Documentation/ceph-pool-crd.md

```diff
@@ -9,21 +9,37 @@ indent: true
 Rook allows creation and customization of storage pools through the custom resource definitions (CRDs). The following settings are available
 for pools.
 
-## Sample
+## Samples
 
+### Replication
+
+For optimal performance while adding redundancy to the system, configure the data to be copied in full to multiple locations.
 ```yaml
 apiVersion: ceph.rook.io/v1beta1
 kind: Pool
 metadata:
-  name: ecpool
+  name: replicapool
   namespace: rook-ceph
 spec:
+  failureDomain: host
   replicated:
-  #  size: 3
+    size: 3
+```
+
+### Erasure Coding
+
+To lower your storage capacity requirements while adding redundancy, use [erasure coding](#erasure-coding).
+```yaml
+apiVersion: ceph.rook.io/v1beta1
+kind: Pool
+metadata:
+  name: ecpool
+  namespace: rook-ceph
+spec:
+  failureDomain: osd
```

Comment:
What is correct? Above it is `failureDomain: host`

Reply:
Either one is correct, depending on the desired configuration. The details are explained below.